app/src/examples/AnimatedSensorGyroscopeExample.tsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import { Pressable, SafeAreaView, StyleSheet, Text, View } from 'react-native';
+import Animated, {
+  SensorType,
+  useAnimatedSensor,
+  useAnimatedStyle,
+  useSharedValue,
+  withSpring,
+} from 'react-native-reanimated';
+
+function clampWorklet(num: number, min: number, max: number) {
```

Comment:
Let's use `clamp` from public API.

Reply:
For now I will leave as it is, and update it in a separate PR as soon as we will introduce this API.